artifactory.py

```diff
@@ -862,6 +883,25 @@ def stat(self, pathobj):
             md5=checksums.get("md5", None),
             is_dir=is_dir,
             children=children,
+            repo=jsn.get("repo", None),
+        )
+
+        return stat
+
+    def download_stats(self, pathobj):
```

Comment:
`download_stats` look like "to download statistic" when it's placed near by `get_stat_json`

Let's call it `get_download_stat`

Reply:
Makes, sense, let me do it

Reply:
I see, it's the way how we call `stats`, it's fine